packages/types/src/codec/Set.ts

```diff
@@ -80,6 +80,24 @@ function decodeSet (setValues: SetValues, value: string[] | Set<string> | Uint8A
   return decodeSetNumber(setValues, value);
 }
 
+const iskeyMap = new Map<string, string>();
+
+function memoizedIskey (value: string): string {
+  {
+    const iskey = iskeyMap.get(value);
+
+    if (iskey) {
+      return iskey;
+    }
+  }
+
+  const iskey = `is${stringUpperFirst(stringCamelCase(value))}`;
+
+  iskeyMap.set(value, iskey);
+
+  return iskey;
+}
```

Comment:
Will drop this - it only gets use inside `with` (and classes are cached, so it really doesn't make a difference, i.e. it is more micro-optimization that adds maintanence)